src/app/components/pill.tsx

```diff
@@ -11,7 +11,7 @@ export function Pill({ hoverLabel, label, ...props }: PillProps) {
     <Tooltip disabled={!hoverLabel} label={hoverLabel} maxWidth="200px" placement="bottom">
       <Box
         border="1px solid"
```

Comment:
Maybe we can update the border also? 

```suggestion
        border="default"
```